src/main/java/meteordevelopment/meteorclient/systems/modules/movement/NoFall.java

```diff
@@ -185,8 +174,6 @@ else if (mode.get() == Mode.Place) {
     }
 
     private void useItem(FindItemResult item, boolean placedWater, BlockPos blockPos, boolean interactItem) {
-        if (!item.found()) return;
```

Comment:
Why?

Reply:
The code has done item search in the previous code.